supervisor/core.py

```diff
@@ -244,6 +244,9 @@ async def start(self):
             self.sys_create_task(self.sys_resolution.fixup.run_autofix())
 
             self.state = CoreState.RUNNING
+            await self.sys_homeassistant.websocket.async_supervisor_update_event(
```

Comment:
let's use the `self.sys_homeassistant.websocket.supervisor_update_event`

Reply:
or like above create a task

Reply:
Why? this is in async context?

Reply:
because on an error, we never execute the function until the end and we are not interested if HA get it or not, at this point it's not our problem